presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java

```diff
@@ -36,7 +36,8 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
 import io.airlift.units.DataSize;
-import javafx.util.Pair;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
unnecessary import?

Reply:
The function returns a Pair - I will change it to return an ImmutablePair